src/main/resources/application.yml

```diff
@@ -15,4 +15,5 @@ server:
 exam-results-transmitter-service:
   session-url: http://localhost:32842
   assessment-url: http://localhost:32841
-  exam-url: http://localhost:8081
\ No newline at end of file
+  exam-url: http://localhost:8081
+  validate-trt-xml: false
```

Comment:
This configuration will be turned to true and be the default when we have all the data necessary to build the TRT.  Consider this a WIP project as we slowly add more pieces to the TRT and the exam code. 

For example, item selection and item scoring still isn't ready in Exam so there is no data to put in the TRT, and validation will fail.  I thought this solution was cleaner rather than having the validator code present but commented out.